### PR TITLE
Add new shapes

### PR DESCRIPTION
**Pull request by Rychu-Peja**

### CODE REVIEW

File: firmware/src/src/AR/oled.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@
   }
 #endif
 
-#define D_S 6 // DIAMOND_SIZE
+#define P_S 6 // POINT_SIZE
```
Comment (mlody444) [inline firmware/src/src/AR/oled.cpp]: Please use full names (example - POINT_SIZE)

Shouldn't point size be total size (currently size is 2*P_S + 1 - 1 as middle point) - to be considered

**Review by mlody444 — CHANGES_REQUESTED** [review]
Minor comment

File: firmware/src/src/AR/oled.cpp (hunk 1 above)
Comment (mlody444) [reply firmware/src/src/AR/oled.cpp]: Full named used

Second comment not addressed, please use #define POINT_HALF_SIZE ((int16_t) POINT_SIZE / 2)

Comment (Rychu-Peja) [reply firmware/src/src/AR/oled.cpp]: Implemented

**Review by mlody444 — APPROVED** [review]

**Pull request merged**